### DISCUSSION TITLE
Playnite Game Export script/built in maybe?

### DISCUSSION THREAD

**Discussion by Her0es** [ClassicOldSong/Apollo#181] [Ideas]
First off, love this fork. Literally streamlines everything I wanted in main Sunshine. Only missing a couple of things. When I used Sunshine, one of the mods there had some handy scripts, like a Playnite game exporter that auto set up art and settings in Sunshine when used. It also set up for auto closing the stream on game exit. Now, I got games closing properly with a 0 sec wait exit timeout when using the same script just for the auto close function. My question is can we get a game export built around Apollo?

**Comment by ClassicOldSong** [comment]
The script for Sunshine still should work, you just need to copy its output to Apollo's folder manually.

**Reply by Her0es** [reply]
I did that and it didn't import correctly. Maybe it was just being wonky?

**Reply by Her0es** [reply]
Update, got it working but it also overwrites my settings for using virtual display when exporting new games. Wish I knew how to script it myself to save that setting for all the games I already exported.

**Reply by ClassicOldSong** [reply]
You need to contact them to support Apollo in this case😂
